src/general/WorldGenerationSettings.cs

```diff
@@ -160,7 +161,6 @@ public string GetTranslatedDifficultyString()
             Localization.Translate("PERCENTAGE_VALUE").FormatSafe(Math.Round(GlucoseDecay * 100, 1)),
             OsmoregulationMultiplier,
             TranslationHelper.TranslateFeatureFlag(FreeGlucoseCloud),
-            TranslationHelper.TranslateFeatureFlag(PassiveGainOfReproductionCompounds),
```

Comment:
As there's fewer parameters passed to the format, the English text for `DIFFICULTY_DETAILS_STRING` needs to be updated to compensate (by removing one placeholder from it).